packages/react-data-grid/src/Cell/cellContentRenderers.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import CellContent from './CellContent';
+import { CellContentRendererProps } from '../common/types';
+
+export function legacyCellContentRenderer<R>(props: CellContentRendererProps<R>) {
+  return React.createElement<CellContentRendererProps<R>>(CellContent, props);
```

Comment:
Any reason not to write it like this `<CellContent {...props} />`?

Reply:
I recently found out that TS will transpile such code like this:
![image](https://user-images.githubusercontent.com/567105/67227849-002bf880-f430-11e9-9014-2db3390199dd.png)

which is less efficient than manually using `React.createElement`:
![image](https://user-images.githubusercontent.com/567105/67227967-39646880-f430-11e9-845e-80ac705c4fa5.png)

Babel handles it better: https://babeljs.io/repl#?code_lz=GYVwdgxgLglg9mABAGwKYHMCGECeBhVZZPBKVMKAJXIBNUAnBgCgAd64WBnASkQG8AUIkSMoIekgA8BIiQrko_AHQq2HTgF9EAegB8AbgEagA